Sources/StreamChatUI/ChatMessageList/Reactions/ChatMessageReactionAuthorsVC/ChatMessageReactionAuthorsVC.swift

```diff
@@ -0,0 +1,185 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// A view controller that renders all the reactions of a message.
+open class ChatMessageReactionAuthorsVC:
+    _ViewController,
+    ThemeProvider,
+    ChatMessageControllerDelegate,
+    UICollectionViewDataSource,
+    UICollectionViewDelegate,
+    UICollectionViewDelegateFlowLayout {
+    /// The message controller of message that the reactions belong.
+    open var messageController: ChatMessageController!
+
+    /// Label that shows how many reactions the message has.
+    open lazy var topLabel: UILabel = UILabel()
+        .withoutAutoresizingMaskConstraints
+        .withBidirectionalLanguagesSupport
+        .withAdjustingFontForContentSizeCategory
+
+    /// `UICollectionViewFlowLayout` instance for the collection view.
+    open lazy var flowLayout: UICollectionViewFlowLayout = ChatMessageReactionAuthorsFlowLayout()
+
+    /// `UICollectionView` instance to display the reaction authors.
+    open lazy var collectionView: UICollectionView = UICollectionView(
+        frame: .zero,
+        collectionViewLayout: flowLayout
+    ).withoutAutoresizingMaskConstraints
+
+    /// The width and height of each reaction author cell.
+    open var reactionAuthorCellSize: CGSize {
+        CGSize(width: 64, height: 110)
+    }
+
+    /// By default it is 10. It will prefetch the reactions before showing the last 10th reaction.
+    open var prefetchThreshold: Int {
+        10
+    }
+
+    /// A Boolean indicating whether the reactions are currently loading.
+    public var isLoadingReactions: Bool = false
```

Comment:
nit: 
```suggestion
    public private(set) var isLoadingReactions: Bool = false
```

Reply:
I'm actually going to put this private, because this might be a temporary API until we do this in the Core SDK